src/AutosizeInput.js

```diff
@@ -32,6 +32,13 @@ const AutosizeInput = React.createClass({
 		this.copyInputStyles();
 		this.updateInputWidth();
 	},
+	shouldComponentUpdate (nextProps) {
+		// if this is the first time
+		if (nextProps === this.props) {
+			return true;
+		}
+		return nextProps.value !== this.props.value;
```

Comment:
this would break component updates if any other props change, which is going to cause a problem :(